contrib/backporting/check-stable

```diff
@@ -185,7 +185,8 @@ gitlib::github_api_token
 
 echo "Checking for backports on branch '${STABLE_BRANCH}'"
 
-stable_prs=($(get_prs "needs-backport/${STABLE_BRANCH}" "backport-done/${STABLE_BRANCH}"))
+# Don't consider PRs with `backport/author`, which the author will backport themselves.
+stable_prs=($(get_prs "needs-backport/${STABLE_BRANCH}" "backport/author" ))
```

Comment:
Hmm, is it OK to remove the `backport-done/*` bit from here?

Reply:
Depends on whether we want this to be resilient against mislabelled PRs. If so, we'd also need to ignore those with `backport-pending/branch`, which we were not currently doing. I'm open to either solution- this is just the minimal change.

Reply:
Ah right, yeah I agree. I'm not sure why we found it necessary to exclude the `backport-done/*` labels in the first place.